main.py

```diff
@@ -0,0 +1,63 @@
+from robyn import Robyn, SubRouter, jsonify
```

Comment:
@DarrenSeng , this file should not exist. 

All this code should be a part of the `base_routes` files instead of what I wrote. 

And the corresponding routes should be tested the way demonstrated!